docs/guides/miscellaneous/how_to_quickly_explore_expectations_in_a_notebook.md

```diff
@@ -2,4 +2,227 @@
 title: How to quickly explore Expectations in a notebook
 ---
 
-This article is a stub.
+Building [Expectations](../../reference/expectations/expectations.md) as you conduct exploratory data analysis is a great way to ensure that your insights about data processes and pipelines remain part of your team's knowledge.
+
+This guide will help you quickly get a taste of Great Expectations, without even setting up a [Data Context](../../reference/data_context.md). All you need is a notebook and some data.
+
+:::note This how-to guide assumes you have already:
+- Installed Great Expectations (e.g. `pip install great_expectations`)
+- Have access to a notebook (e.g. `jupyter notebook`, `jupyter lab`, etc.)
+- Obtained a sample of data to use for exploration
+:::
```

Comment:
Please utilize the `<Prerequisites>` tag.

Reply:
This one is slightly different from the default re-reqs since it's explicitly *not* using a data context, so I'd keep this as-is.

Reply:
Makes sense to me! I'll resolve this conversation.